### PR TITLE
web/satellite:access grant date picker

### PR DESCRIPTION
**Pull request by mbouzi**
What: Date Picker component creation for Access Permissions Step. Also added duration dropdown and added picker component to permissions step

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/accessGrants/permissions/DurationPicker.vue [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+        &__input {
+            display: none;
+        }
+    }
+}
+
+</style>
+
+<style lang="scss">
```
Comment (mbouzi) [inline web/satellite/src/components/accessGrants/permissions/DurationPicker.vue:110]: picker popup was out of scope

File: web/satellite/src/components/accessGrants/steps/PermissionsStep.vue [hunk 2]
```diff
@@ -28,7 +28,13 @@
                 </div>
             </div>
             <div class="permissions__content__right">
+				<div class="permissions__content__right__duration-select">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/steps/PermissionsStep.vue:31]: indents area broken

File: web/satellite/src/components/accessGrants/permissions/DurationSelection.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div class="duration-selection">
+		<div
+            class="duration-selection__toggle-container"
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/permissions/DurationSelection.vue:7]: indents

File: web/satellite/src/components/accessGrants/permissions/DurationPicker.vue [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+			<ul class="duration-picker__list__column">
+				<li class="duration-picker__list__column-item">6 Months</li>
+				<li class="duration-picker__list__column-item">1 Week</li>
+				<li class="duration-picker__list__column-item">1 Year</li>
+			</ul>
+		</div>
+		<hr class="duration-picker__break">
+		<div class="duration-picker__date-picker__wrapper">
+			<DatePicker
+			range
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/permissions/DurationPicker.vue:21]: add 4-space indentations for attributes here

File: web/satellite/package.json [hunk 5]
```diff
@@ -25,6 +25,7 @@
     "vue-clipboard2": "0.3.1",
     "vue-property-decorator": "9.0.0",
     "vue-router": "3.4.3",
+    "vue2-datepicker": "^3.7.0",
```
Comment (VitaliiShpital) [inline web/satellite/package.json:28]: remove ^ here

File: web/satellite/src/components/accessGrants/permissions/BucketsSelection.vue [hunk 6]
```diff
@@ -4,8 +4,8 @@
 <template>
     <div class="buckets-selection">
         <div
-            class="buckets-selection__toggle-container"
-            @click.stop="toggleDropdown"
+			class="buckets-selection__toggle-container"
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/permissions/BucketsSelection.vue:7]: indents

File: web/satellite/src/components/accessGrants/permissions/DurationSelection.vue [hunk 7]
```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div class="duration-selection">
+		<div
+        	class="duration-selection__toggle-container"
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/permissions/DurationSelection.vue:7]: indents

File: web/satellite/src/components/accessGrants/steps/PermissionsStep.vue [hunk 8]
```diff
@@ -28,7 +28,13 @@
                 </div>
             </div>
             <div class="permissions__content__right">
+		        <div class="permissions__content__right__duration-select">
+					<p class="permissions__content__right__duration-select__label">Duration</p>
+					<DurationSelection />
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/steps/PermissionsStep.vue:33]: indents

**Review by VitaliiShpital — DISMISSED** [review]

**Review by VitaliiShpital — DISMISSED** [review]

**Review by VitaliiShpital — APPROVED** [review]

**Review by crawter — APPROVED** [review]

**Pull request merged**